practice/src/js/constants/url.js

```diff
@@ -2,4 +2,4 @@ const URLS = {
   HOME: 'index.html',
 };
 
-export default URLS;
+export { URLS };
```

Comment:
@JennyDoJD why should we need the update?

Reply:
i think if we later want to add new properties to the URLS object, it will keep the source code clearer when we need multiple variable names or object names from the same module